drivers/rawexec/driver_test.go

```diff
@@ -237,103 +236,6 @@ func TestRawExecDriver_StartWait(t *testing.T) {
 	require.NoError(harness.DestroyTask(task.ID, true))
 }
 
-func TestRawExecDriver_StartWaitRecoverWaitStop(t *testing.T) {
```

Comment:
Note to reviewers: this is moved to `driver_unix_test.go` unchanged, including any modernization to use `shoenig/test`, `t.Cleanup`, etc. I'll take a second pass over these files in a separate refactoring PR to do all that.